src/Strategy/Maven/Plugin.hs

```diff
@@ -107,28 +192,72 @@ mavenPluginDependenciesCmd plugin =
     { cmdName = "mvn"
     , cmdArgs =
         [ group plugin <> ":" <> artifact plugin <> ":" <> version plugin <> ":aggregate"
-        , "-DgraphFormat=json"
+        , "-DgraphFormat=text"
         , "-DmergeScopes"
         , "-DreduceEdges=false"
+        , "-DshowVersions=true"
+        , "-DshowGroupIds=true"
+        , "-DshowOptional=true"
+        , "-DrepeatTransitiveDependenciesInTextGraph=true"
         ]
     , cmdAllowErr = Never
     }
 
+mavenPluginReactorCmd :: DepGraphPlugin -> Command
+mavenPluginReactorCmd plugin =
+  Command
+    { cmdName = "mvn"
+    , cmdArgs =
+        [ group plugin <> ":" <> artifact plugin <> ":" <> version plugin <> ":reactor"
+        , "-DgraphFormat=json"
+        , "-DoutputFileName=reactor-graph.json"
```

Comment:
[Optional] might be better to add `fossa-` prefix in the name.

Reply:
that's a good idea, thanks!